### PR TITLE
feat(smoothieware): Add OT Smoothieware to CI

### PR DESCRIPTION
**Pull request by Laura-Danielle**
## Overview
This PR adds the smoothie repository to Travis-CI. Travis will attempt to compile smoothie and deploy the updated .hex file to AWS. Closes https://github.com/Opentrons/opentrons/issues/3301

## Changelog
- Edit pull/issue templates to match other opentrons repos
- Edit README with info specific to OT smoothieware
- Edit make file to add a `build` parameter and some comments
- Edit travis yml to deploy/ensure correct compiler is being used

## Review Requests
- Anything to change in the makefile? Any requested changes to travis yml
- Check that the hex file is automatically uploaded to AWS bucket

## TO-DO (in future)
- Add tests to CI for smoothie
- Add a `setup` env parameter for the makefile to automatically install the correct compiler

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!--
+  Thanks for taking the time to file an issue! Please make sure you've read the
+  "Opening Issues" section of our Contributing Guide:
+
+  https://github.com/Opentrons/opentrons/blob/edge/CONTRIBUTING.md#opening-issues
```
Comment (mcous) [inline .github/ISSUE_TEMPLATE.md:5]: Is pointing users to the monorepo contributing guide here intentional?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!--
+  Thanks for taking the time to open a pull request! Please make sure you've
+  read the "Opening Pull Requests" section of our Contributing Guide:
+
+  https://github.com/Opentrons/opentrons/blob/edge/CONTRIBUTING.md#opening-pull-requests
```
Comment (mcous) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Ditto

File: .github/semantic.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# always validate the PR title, and ignore the commits
+# https://github.com/probot/semantic-pull-requests
+titleOnly: true
```
Comment (mcous) [inline .github/semantic.yml:3]: Good call! I just enabled the PR bot on this repository

**Review by mcous — APPROVED** [review]
🤖

File: README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+For more information on smoothie as a whole, please check out their documentation [here](http://smoothieware.org/)
+
+### Materials
+Opentrons utilizes the `LPC1768` chip
+
+### Quick Start
+These are the quick steps to get OT Smoothie dependencies installed on your computer:
+* Pull down a clone of the Opentrons Smoothie github fork project to your local machine.
+* Install [GNU-ARM compiler](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads) for your Operating System.
+* Ensure that your path is correctly set by searching for the location of: `arm-none-eabi-gcc/../../bin`
```
Comment (sfoster1) [inline README.md:18]: Not sure what that path is?

**Review by sfoster1 — APPROVED** [review]
Looks good, looks like the builds work.

File: README.md (hunk 4 above)
Comment (Laura-Danielle) [reply README.md:18]: Trying my best to do an arbitrary path to the gcc bin directory 🙃

**Pull request merged**